lib/screens/home/search/pet_sitter/pet_sitter_profile_page.dart

```diff
@@ -0,0 +1,112 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/widgets.dart';
+import '../../../../model/user.dart';
```

Comment:
Can you use the package path please?
It will crash if we change any file in this relative hierarchy without compiler warnings.